java-tools/OsmAndServer/src/main/resources/application.yml

```diff
@@ -56,10 +56,12 @@ mapillary:
 osmlive:
   status: /var/www-download/api/.proc_timestamp
 
-
 monitoring:
   enabled: ${ENABLE_MONITORING:false}
 
+motd:
```

Comment:
settings is not a parameter it self and should use standard location /var/www-download/ for all settings, configs etc. So we are not going to pollute yml with 10-20 same locations